skyvern/forge/sdk/prompting.py

```diff
@@ -98,7 +98,7 @@ def load_prompt(self, template: str, **kwargs: Any) -> str:
             str: The populated template.
         """
         try:
-            template = os.path.join(self.model, template)
+            template = '/'.join([self.model, template])
```

Comment:
```suggestion
            template = "/".join([self.model, template])
```

our linter suggests double quotes